js/data.js

```diff
@@ -6,3 +6,39 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var previousEntriesJSON = localStorage.getItem('journalEntry');
+if (previousEntriesJSON !== null) {
+  data = JSON.parse(previousEntriesJSON);
+}
+
+var form = document.querySelector('#form');
+var title = document.querySelector('#title-text');
+var urlInput = document.querySelector('#input-url');
+var notes = document.querySelector('#notes-text');
+var photo = document.querySelector('#main-image');
+
+function submitListener(event) {
+  event.preventDefault();
+  data.entries.unshift({
+    title: title.value,
+    photoURL: urlInput.value,
+    notes: notes.value,
+    entryID: data.nextEntryId
+  });
+  data.nextEntryId++;
+
+  photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  title.value = '';
+  urlInput.value = '';
+  notes.value = '';
+}
+
+form.addEventListener('submit', submitListener);
```

Comment:
This should be in the `main.js` file. `data.js` is only for the data model and localStorage code.

Reply:
This was my mistake for misunderstanding your earlier feedback.